yt_dlp/extractor/gettr.py

```diff
@@ -108,3 +109,70 @@ def _real_extract(self, url):
             'duration': float_or_none(post_data.get('vid_dur')),
             'tags': post_data.get('htgs'),
         }
+
+
+class GettrStreamingIE(GettrIE):
+    IE_NAME = 'Gettr:streaming'
```

Comment:
`site:type` IE_NAME format usually doesn't have capital letters
since GettrIE doesn't set it's own IE_NAME either,
```suggestion
```